packages/common-scripts/src/omnilock-bitcoin.ts

```diff
@@ -0,0 +1,93 @@
+import { bytes } from "@ckb-lumos/codec";
+import { bech32 } from "bech32";
+import bs58 from "bs58";
+
+export function decodeAddress(address: string): ArrayLike<number> {
+  try {
+    // Bech32
+    if (address.startsWith("bc1")) {
```

Comment:
the taproot address is encoded via bech32m so it will fail when trying to decode via bech32, the following catch block will throw an error to tell the user that the p2tr is not supported yet

Reply:
the taproot address is encoded via bech32m so it will fail when trying to decode via bech32, the following catch block will throw an error to tell the user that the p2tr is not supported yet